app/js/components/common/savedResultsTable.js

```diff
@@ -28,11 +29,13 @@ const SavedResultsTable = ({
           </thead>
           <tbody>
             {
-              results.map((result, index) => {
-                const { uuid, description } = result;
-                return (
+              results.map((result) => {
+                if (result.name != 'appointmentschedulingui.appointmentDataSetDefinition.dailyAppointments') {
```

Comment:
@dkayiwa please do you know what this (`appointmentschedulingui.appointmentDataSetDefinition.dailyAppointments`) is?